README.md

```diff
@@ -1,15 +1,122 @@
+cd config/manager && /home/mgoerens/dev/github.com/opdev/synapse-operator/bin/kustomize edit set image controller=controller:latest
+/home/mgoerens/dev/github.com/opdev/synapse-operator/bin/kustomize build config/default | kubectl apply -f -
+namespace/synapse-operator-system created
+customresourcedefinition.apiextensions.k8s.io/synapses.synapse.opdev.io configured
+serviceaccount/synapse-operator-controller-manager created
+role.rbac.authorization.k8s.io/synapse-operator-leader-election-role created
+clusterrole.rbac.authorization.k8s.io/synapse-operator-manager-role created
+clusterrole.rbac.authorization.k8s.io/synapse-operator-metrics-reader created
+clusterrole.rbac.authorization.k8s.io/synapse-operator-proxy-role created
+rolebinding.rbac.authorization.k8s.io/synapse-operator-leader-election-rolebinding created
+clusterrolebinding.rbac.authorization.k8s.io/synapse-operator-manager-rolebinding created
+clusterrolebinding.rbac.authorization.k8s.io/synapse-operator-proxy-rolebinding created
+configmap/synapse-operator-manager-config created
+service/synapse-operator-controller-manager-metrics-service created
+deployment.apps/synapse-operator-controller-manager created
+```
+
+This creates a dedicated namespace `synapse-operator-system` and all required
+resources (including the CRD) for the controller to run.
+
+To cleanup all resources:
+
+```shell
+$ make undeploy
+/home/mgoerens/dev/github.com/opdev/synapse-operator/bin/kustomize build config/default | kubectl delete -f -
+namespace "synapse-operator-system" deleted
+customresourcedefinition.apiextensions.k8s.io "synapses.synapse.opdev.io" deleted
+serviceaccount "synapse-operator-controller-manager" deleted
+role.rbac.authorization.k8s.io "synapse-operator-leader-election-role" deleted
+clusterrole.rbac.authorization.k8s.io "synapse-operator-manager-role" deleted
+clusterrole.rbac.authorization.k8s.io "synapse-operator-metrics-reader" deleted
+clusterrole.rbac.authorization.k8s.io "synapse-operator-proxy-role" deleted
+rolebinding.rbac.authorization.k8s.io "synapse-operator-leader-election-rolebinding" deleted
+clusterrolebinding.rbac.authorization.k8s.io "synapse-operator-manager-rolebinding" deleted
+clusterrolebinding.rbac.authorization.k8s.io "synapse-operator-proxy-rolebinding" deleted
+configmap "synapse-operator-manager-config" deleted
+service "synapse-operator-controller-manager-metrics-service" deleted
+deployment.apps "synapse-operator-controller-manager" deleted
+```
+
+### Install via OLM
+
+The Synapse operator has not been published on the Operator Hub yet.
 
 ## Notes and pre-requisites
 
 - The [postgres-operator](https://github.com/CrunchyData/postgres-operator)
   needs to be installed. Specifically the `PostgresCluster` CRD is required.
+  This is only required if you intend to deploy a PostgreSQL instance alongside
+  Synapse.
 - Tested on OpenShift 4.9.0
 
-# Related Links
+# Related links
 
```

Comment:
I'm also missing links to the examples. It's kind of obvious for people looking at the project since the folder is on the root of the project but I feel that would be nice to have a link on the main README as well point to the examples README maybe.

Reply:
Ah thanks ! Yes, I wanted to add it, and just forgot about it. Will do